containers/documentation.docker/build_documentation

```diff
@@ -131,6 +138,21 @@ GITBOOK_ARGS=(--gitbook "${INSTALLED_GITBOOK_VERSION}")
 . "$script_dir/buildLib" || ferr "failed to source buildLib"
 
 
+################################################################################
+### HELPER FUNCTIONS ###########################################################
+################################################################################
+
+# Pre-process path for call to e.g. Python or Node.js
+function prepare_path() {
+    local any_path="$1"
+    # Convert to Windows path, e.g. '/cygwin/c/...' to 'C:\...'
+    if [ "$is_cygwin" = true ]; then
```

Comment:
if $is_cygwin; then

Reply:
I think it is better to stick to `$is_cygwin` because it is easy to read. I am not in favor of defensive programming. If somebody works on the code who should not it will break anyway. Super safe if conditions will not help much.

Reply:
Please remove the leading `function`. It should probably look like this:

```
expand_cygpath() {
  for arg in "$@"; do
    if type cygpath &> /dev/null ; then
      echo "$(cygpath "$arg")"
    else
      echo "$arg"
    fi
  done
}
```